.github/workflows/release.yml

```diff
@@ -1,15 +1,25 @@
 name: Auto-release on PR merge
 
 on:
-  # ATM, this is the closest trigger to a PR merging
-  push:
+  # This action should be run in workflows triggered by `pull_request_target`
+  # (not by regular `pull_request`!)
+  pull_request_target:
     branches:
+      # Create a release whenever a PR is merged into one of these branches:
       - master
+    types:
+      - closed
+  # Allow manually triggering a release via a "Run workflow" button on the
+  # workflow's page:
+  workflow_dispatch:
```

Comment:
I don't think this will work, unless there happens to be an unreleased yet merged PR with a "release" label.

Reply:
so if I retroactively tag most recent PR with `release` and dispatch the action?  I think that already would be useful. I just would need to add a note on that. Might even try here (by removing `release` tag first).

Reply:
That might work.

Reply:
ok, added a comment... we will see